lib/repl.js

```diff
@@ -521,7 +515,9 @@ REPLServer.prototype.complete = function(line, callback) {
                 }
               }
             }
-          } catch (e) {}
+          } catch (e) {
+            // ignore
+          }
```

Comment:
A comment in an empty block passes the `no-empty` rule, http://eslint.org/docs/rules/no-empty.html. I think while a comment like this won't hurt, it may be hard for someone to discover that they can circumvent the rule like this, so I think we shouuld disable that rule.